### DISCUSSION TITLE
Does Nginx Unit support SSI (Server-Side-Includes)?

### DISCUSSION THREAD

**Discussion by kvelaro** [nginx/unit#1322] [Support] [answered]
How to turn SSI feature on? How to set flag when compiling?

Now our build is not aware of SSI, thats why we have technical message printed

`"serverTime": "<!--#config timefmt='%s'--><!--# echo var='date_local'-->"`

**Comment by lcrilly** [comment, marked as answer]
Unit does not support SSI.

At face value, it seems like an unlikely feature given the primary use case of running application code, so producing content at runtime is something the application code can do.

Are you using Unit entirely for serving static content (web server) or do you intend to use SSI alongside application code?

**Reply by kvelaro** [reply]
Thank you for dispelling my doubts.

Yes, we use SSI alongside application code. (Nginx + Nginx)

As a workaround: Nginx with SSI feature set on must be on top of Unit (Nginx + Unit). What do you say @lcrilly?

The source of problem appeared after we made docker image(Unit) with our api (which uses SSI) locally (without external Nginx).

**Reply by lcrilly** [reply]
Your workaround is currently the best approach.

There is a previous discussion about SSI here (https://github.com/nginx/unit/issues/838) - it is something we hope to provide with JavaScript-based response filtering, but not currently planned.

**Reply by kvelaro** [reply]
Thanks a lot, will be waiting
